### PR TITLE
201_replace_heatmap_with_relevance

### PR DESCRIPTION
**Pull request by elboyran**
#201

### CODE REVIEW

File: tutorials/lime_text.ipynb [hunk 1]
```diff
@@ -180,8 +180,8 @@
    ],
```
Comment (cwmeijer) [inline tutorials/lime_text.ipynb:180]: <p>Much better already.</p><p>"while words against the selected class - in blue." I think would read even easier if it were:</p><p>"while words against the selected class are highlighted in blue."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

File: tutorials/rise_imagenet.ipynb [hunk 2]
```diff
@@ -7,7 +7,7 @@
    "source": [
```
Comment (cwmeijer) [inline tutorials/rise_imagenet.ipynb:7]: <p>Reads well :-)</p><p>Formatting: there are 3 double spaces in the second paragraph of this block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

File: tutorials/rise_mnist.ipynb [hunk 3]
```diff
@@ -7,9 +7,9 @@
    "source": [
```
Comment (cwmeijer) [inline tutorials/rise_mnist.ipynb:7]: <p>In the previous notebook you used 'relevance scores', which you are using in the rest of this notebook, too. Let's get rid of the 'attributions', even though a great term by itself, for consistency sake as to not confuse the user.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

File: tutorials/rise_text.ipynb [hunk 4]
```diff
@@ -8,7 +8,7 @@
     "## Interpreting a movie review sentiment model with RISE\n",
```
Comment (cwmeijer) [inline tutorials/rise_text.ipynb:8]: <p>I would like to add a comma after 'text' like so:</p><p>DIANNA includes a visualization package, capable of highlighting each word of a text,<strong> </strong>based on their relevance scores.</p><p>In my mind I parsed it like 'text-based' the first 2 times I read the sentence.</p><p> </p><p>Also, there is a double space in 'in  red'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (cwmeijer) [inline tutorials/rise_text.ipynb:8]: <p>This has nothing to do with this PR, but I'm not happy with this example text for a couple of reasons:</p><ul><li>It is merely positive. As an example we could better have used something with positive and negative elements.</li><li>The short sentence contains a word that both I, and a colleague, didn't know 'delectable'. Not ideal for an example.</li></ul><p>I'll add an issue about this, as it concerns more notebooks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Review by cwmeijer — CHANGES_REQUESTED** [review]
Great work. I left some comments for textual improvements.
I also complained about the text example, but that's not really part of this PR so I suggest that you leave that for now as it is and get this merged :-)

**Comment by egpbos** [comment]
Agree with @cwmeijer's comments, great work.

There are a couple of mentions left. I see 56 results in 12 files for the search term "heatmap". Most of these are in code, two in CONTEXT.md. I would say the code ones should be replaced mostly as well, to stay consistent (i.e. have self-documenting code which is in sync with the actual documentation). Notable exception could be dianna/visualization/image.py, where indeed we are plotting the relevance scores as heatmaps :) Still, it could make more sense in our package to also call the input parameter relevance score as well (we are not a plotting package which does generic heatmap plotting, that's what matplotlib is for). In that case, it would make sense to mention in the docstring that the output is a heatmap.

**Comment by elboyran** [comment]
> Agree with @cwmeijer's comments, great work.
> 
> There are a couple of mentions left. I see 56 results in 12 files for the search term "heatmap". Most of these are in code, two in CONTEXT.md. I would say the code ones should be replaced mostly as well, to stay consistent (i.e. have self-documenting code which is in sync with the actual documentation). Notable exception could be dianna/visualization/image.py, where indeed we are plotting the relevance scores as heatmaps :) Still, it could make more sense in our package to also call the input parameter relevance score as well (we are not a plotting package which does generic heatmap plotting, that's what matplotlib is for). In that case, it would make sense to mention in the docstring that the output is a heatmap.

We need a new issue for in the code if we decide to change it.

**Comment by egpbos** [comment]
Makes sense, I can make it. Then just two mentions in CONTEXT.md remaining :)

**Comment by elboyran** [comment]
> Great work. I left some comments for textual improvements. I also complained about the text example, but that's not really part of this PR so I suggest that you leave that for now as it is and get this merged :-)

Thanks. Could you, suggest the textual improvements as direct changes?

**Comment by elboyran** [comment]
> Makes sense, I can make it. Then just two mentions in CONTEXT.md remaining :)

Fixed.

**Pull request merged**